tests/test_documentation.py

```diff
@@ -0,0 +1,58 @@
+from inspect import isfunction, isclass, getmembers, ismodule, ismethod, signature, getmodule
+import importlib
+from itertools import compress
+import pytest
+
+from keras.utils.test_utils import keras_test
+
+modules = ['keras.layers', 'keras.models', 'keras', 'keras.backend.tensorflow_backend']
+accepted_name = ["from_config"]
+accepted_module = ["keras.legacy.layers", "keras.utils.generic_utils"]
+
+
+def handle_class(name, member):
+    if name in accepted_name or member.__module__ in accepted_module:
+        return
+    assert member.__doc__ is not None, "class doesn't have any documentation {} {}".format(name, member.__module__, getmodule(member).__file__)
```

Comment:
Lines are in general too long, limit line length to something reasonable (e.g. 80 characters).

Use `Exception` subclasses (e.g. `ValueError`) instead of `assert`.